x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/stats.tsx

```diff
@@ -120,17 +120,18 @@ export function MobileStats({
       trendShape: MetricTrendShape.Area,
     },
     {
-      color: euiTheme.eui.euiColorDisabled,
+      color: euiTheme.eui.euiColorLightestShade,
       title: i18n.translate('xpack.apm.mobile.metrics.load.time', {
-        defaultMessage: 'Slowest App load time',
-      }),
-      subtitle: i18n.translate('xpack.apm.mobile.coming.soon', {
-        defaultMessage: 'Coming Soon',
+        defaultMessage: 'Average App load time',
```

Comment:
can you please share more insights that led us to switch from the slowest to the average app load time?

Reply:
nit: I checked also the mockups and the App is capitalized, isn't it better to have it `Average app load time` ?

Reply:
> can you please share more insights that led us to switch from the slowest to the average app load time?

During last week's mobile call we decided that the "slowest" approach would depend too much on each device's hardware capabilities, which at least for Android are too varied, potentially leading to confusing results. That's my understanding, although probably @akhileshpok might have more details on it.

Reply:
> nit: I checked also the mockups and the App is capitalized, isn't it better to have it `Average app load time` ?

Sounds good! I'll make the changes

Reply:
thank you both for the insights :)